pkgs/misc/emulators/yuzu/default.nix

```diff
@@ -16,7 +16,7 @@ in {
   };
   early-access = libsForQt5.callPackage ./base.nix rec {
     pname = "yuzu-ea";
-    version = "2156";
+    version = "2397";
     branchName = branch;
     src = fetchFromGitHub {
```

Comment:
Same as above

Reply:
Oh, sorry, I must have missed that. That also explains, why the EA version built successfully.